shared/Views/Shared/Components/CourseDetails/Default.cshtml

```diff
@@ -140,16 +126,12 @@
 
           @await Html.PartialAsync("_AboutSchools.cshtml")
 
-          @if (Model.Course.IsSalaried)
-          {
-            @await Html.PartialAsync("_Salary.cshtml")
-          }
-          else
-          {
-            @await Html.PartialAsync("_Fees.cshtml")
-          }
-
-          @await Html.PartialAsync("FinancialSupport/_FinancialSupport.cshtml")
+          <div class="govuk-!-margin-bottom-8">
+            <h2 class="govuk-heading-l" id="section-financial-support">Financial support</h2>
+            <div class="govuk-inset-text">
+              <p class="govuk-body">The government will soon announce the financial incentives for upcoming courses in 2020/21</p>
```

Comment:
Bursaries, scholarships and financial support for 2020/21 will be announced soon.
```suggestion
              <p class="govuk-body">Bursaries, scholarships and financial support for 2020/21 will be announced soon.</p>
```